opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -129,6 +138,9 @@ namespace {
         1, /* gas-oil ratio */
```

Comment:
> if we have to scale for field

We do, simply since the numerator and denominator of the unit are different and therefore different from 1. This is a good example of a "fun" case where the *dimension* is 1 but the *unit* is not (I did not believe such weirdness would ever happen until I saw eclipse decks with field units...)

But I wonder about this: does not the conversion of input depend on these factors as well? And judging from being able to run the SPE cases well we do that conversion correctly.